src/shared/constants.js

```diff
@@ -0,0 +1,74 @@
+export const baseURL = 'https://json-server-for-dims.herokuapp.com';
+export const baseLocalURL = 'http://localhost:3001';
```

Comment:
keep endpoints in `.env` file like other variables, you don't need to commit them

Reply:
```suggestion
const baseURL = process.env.REACT_APP_BASE_URL;
```